rules/cross-platform/impact_hosts_file_modified.toml

```diff
@@ -22,7 +22,7 @@ references = ["https://www.elastic.co/guide/en/beats/auditbeat/current/auditbeat
 risk_score = 47
 rule_id = "9c260313-c811-4ec8-ab89-8f6530e0246c"
 severity = "medium"
-tags = ["Elastic", "Linux", "Windows", "macOS"]
+tags = ["Elastic", "Linux", "Windows", "MacOS"]
```

Comment:
hold up. i think the original is way more correct.
just ask Apple

Reply:
wfm - so long as we are consistent